### ISSUE TITLE
Unnecessary comment edit

### ISSUE THREAD

**Issue by adambabik** [status-im/status-github-bot#18]
Bot edited [this comment](https://github.com/status-im/status-go/issues/661#issuecomment-366527653) but I am not really sure why and what was changed. It seems like unnecessary change.

**Comment by martinklepsch** [comment]
I think the reason for the edit might be "rich link previews" as added to this comment: https://github.com/status-im/open-bounty/issues/304#issuecomment-366421783

**Comment by PombeirP** [comment]
@adambabik yeah, this is the unfurl functionality (didn't seem to work well at all for that link unfortunately)

**Comment by adambabik** [comment]
So, it's probably a bug in https://github.com/probot/unfurl?

**Comment by PombeirP** [comment]
Yeah, I'd say so.
